src/khoj/interface/web/chat.html

```diff
@@ -838,28 +836,24 @@
             fileInput.click();
         }
 
-        function uploadDataForIndexing(file) {
+        function uploadDataForIndexing(files) {
             var dropzone = document.getElementById('chat-body');
 
-            if (!file || (!allowedExtensions.includes(file.type) && !allowedFileEndings.includes(file.name.split('.').pop()))) {
-                dropzone.classList.remove('dragover');
-                if (file) {
-                    alert("Sorry, that file type is not yet supported");
-                }
-                var overlayText = document.getElementById("dropzone-overlay");
-                if (overlayText != null) {
-                    overlayText.remove();
+            for (let file of files) {
+                if (!file || (!allowedExtensions.includes(file.type) && !allowedFileEndings.includes(file.name.split('.').pop()))) {
+                    dropzone.classList.remove('dragover');
+                    if (file) {
+                        alert("Sorry, that file type is not yet supported");
```

Comment:
This would cause the entire upload to be skipped if there's even one unsupported file. Maybe change this so that all the problematic files are aggregated into a list and then included in the alert message, so the user can remove them and try again? Or we can just skip the problematic files, and provide an alert saying which files are going to be skipped. Your call.